doc/userguide/capture-hardware/napatech.rst

```diff
@@ -239,20 +239,131 @@ Now you are ready to start suricata::
 It is possible to specify much more elaborate configurations using this option. Simply by
 creating the appropriate NTPL file and attaching suricata to the streams.
 
+Bypassing Flows
+---------------
+
+On flow-aware Napatech products traffic from individual flows can be automatically 
+dropped or forwarded by the hardware after an initial inspection of the initial 
+packet(s) of the flow by Suricata.  This will save CPU cycles since Suricata 
+does not process packets for a flow that has already been ajudicated. 
+
+This is enabled via the hardware-bypass option in the napatech section of the 
+conf file. When hardware bypass is used it is important that the ports accepting 
+upstream and downstream traffic from the network are configured as a pair so that 
+the hardware can properly process traffic in both directions.  The port pairings 
+are indicated in the "ports" section as a hyphen separated list of port-pairs 
+for each network segment.  E.g.:: 
+
+    napatech:
+      hardware-bypass: true
+      ports[0-1,2-3]
```

Comment:
I added a section in napatech.rst explaining why port-pairs are required even for IDS.  It might make sense to add a separate configuration option where this is not necessary.  However, this will take both more hardware resources and more instruction cycles.   

If I'm going to do this it would be best to add that as a separate feature request once these changes are check in.